packages/frontend/engine/src/assets/map_loader.test.ts

```
@@ -1805,6 +1806,277 @@ describe('C-379 AC-9 — flip flags + GID convention + multi-tileset layers', ()
+      }
+      return null;
+    }) as AssetTagResolver;
+
+    const releaseUrl = mock((_url: string) => {});
+
+    const fetcher = mock((_url: string) => {
+      return Promise.resolve({
+        ok: true,
+        text: () => Promise.resolve(testJtonSource),
+      } as Response);
+    }) as unknown as typeof fetch;
```

Biome (2.5.7)

[error] The body of this arrow function contains a single return statement.

(lint/style/useConsistentArrowReturn)